doc/languages-frameworks/go.section.md

```diff
@@ -62,6 +62,82 @@ The following is an example expression using `buildGoModule`:
 }
 ```
 
+### Obtaining and overriding `vendorHash` for `buildGoModule` {#buildGoModule-vendorHash}
+
+`nix-prefetch` can be used to obtain the actual hash. The following command gets the value of `vendorHash` for package `pet`:
+
+```sh
+cd path/to/nixpkgs
+nix-prefetch -E "{ sha256 }: ((import ./. { }).my-package.overrideAttrs (_: { vendorHash = sha256; })).go-modules"
+```
+
+To obtain the hash without external tools, set `vendorHash = lib.fakeSha256;` and run the build. ([more details here](#sec-source-hashes)).
+
+`vendorHash` can be overridden with `overrideAttrs`. Override the above example like this:
+
+```nix
+{
+  pet_0_4_0 = pet.overrideAttrs (
+    finalAttrs: previousAttrs: {
+      version = "0.4.0";
+
+      src = fetchFromGitHub {
+        inherit (previousAttrs.src) owner repo;
+        rev = "v${finalAttrs.version}";
+        hash = "sha256-gVTpzmXekQxGMucDKskGi+e+34nJwwsXwvQTjRO6Gdg=";
+      };
+
+      vendorHash = "sha256-dUvp7FEW09V0xMuhewPGw3TuAic/sD7xyXEYviZ2Ivs=";
+    }
+  );
```

Comment:
Is this how nixfmt formats it? I thought this is the standard:

```suggestion
  pet_0_4_0 = pet.overrideAttrs (finalAttrs: previousAttrs: {
    version = "0.4.0";
    src = fetchFromGitHub {
      inherit (previousAttrs.src) owner repo;
      rev = "v${finalAttrs.version}";
      hash = "sha256-gVTpzmXekQxGMucDKskGi+e+34nJwwsXwvQTjRO6Gdg=";
    };
    vendorHash = "sha256-dUvp7FEW09V0xMuhewPGw3TuAic/sD7xyXEYviZ2Ivs=";
  });
```

Reply:
> Is this how nixfmt formats it?

Yep. Not sure why it becomes so vertical.

Reply:
Addressed.

Reply:
> Addressed.

Hmm maybe the docs should comply to the nixfmt (even though I don't like it 😿 )...

Reply:
Now, they are all formatted.